mscore/file.cpp

```diff
@@ -792,9 +792,14 @@ QStringList MuseScore::getOpenScoreNames(const QString& filter, const QString& t
       {
       QSettings settings;
       QString dir = settings.value("lastOpenPath", preferences.myScoresPath).toString();
+
       if (preferences.nativeDialogs) {
-            return QFileDialog::getOpenFileNames(this,
-               title, dir, filter);
+            QStringList FileList;
```

Comment:
Sorry! my fault, 
usually i never use capital for local variable. 
Shame on me!